### PR TITLE
Update devnet addresses after re-deployment

### PR DESCRIPTION
**Pull request by nmlinaric**

### CODE REVIEW

File: config.dev.json [hunk 1]
```diff
@@ -1,54 +1,49 @@
...
-          "address": "0xFf6312a3D0022F75d2Bd92CF250B14A5Ea915945",
-          "type": "basic"
         }
       ],
       "resources": [
         {
           "resourceId": "0x0000000000000000000000000000000000000000000000000000000000000000",
           "type": "erc20",
           "address": "0x3D151A97A446C9ea6893038e7C0db73466f3f3af",
-          "burnable": true,
```
Comment (MakMuftic) [inline config.dev.json:51]: why is this removed?

**Review by MakMuftic — CHANGES_REQUESTED** [review]

**Review by MakMuftic — APPROVED** [review]

**Review by mpetrun5 — APPROVED** [review]

**Review by mpetrun5 — APPROVED** [review]
